Program.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections.Generic;

```

Comment:
```suggestion
﻿using System.Collections.Generic;
﻿
```
Отступы между `using`, методами, свойствами и классами должны быть.